### ISSUE TITLE
Labels for Form Inputs

### ISSUE THREAD

**Issue by haydnba** [fac-14/team4week1#17]
https://github.com/fac-14/team4week1/blob/3920c440a6050131acd950fac99ac6ea9a7db005/index.html#L90-L113

Well done for adding the labels to the form inputs - this is really important for accessibility. 
![screen shot 2018-07-06 at 12 17 26](https://user-images.githubusercontent.com/22300973/42371108-283fae9e-8117-11e8-9277-90b49a89858f.png)
Unfortunately you have failed the accessibility audit on this issue because of a syntax error:
`<label for 'first-name'>Your first name</label>`
This should be:
`<label for='first-name'>Your first name</label>`
You missed the = symbol!

🤓

**Comment by dupreesi** [comment]
thanks guys!
